.circleci/config.yml

```diff
@@ -607,6 +647,147 @@ jobs:
           key: influxdb-cross-build-{{ .Branch }}-{{ .Revision }}
           paths:
             - /tmp/go-cache
+      - persist_to_workspace:
+          root: .
+          paths:
+            - artifacts/*amd64.deb
+
+  # Note: command-level insertion of AWS environment variables is required to
+  # shadow separate legacy credentials used for dl.influxdata.com s3 bucket in
+  # other CI for this repo
+  # see the following known issue for more info: https://support.circleci.com/hc/en-us/articles/360021415793-Wrong-AWS-credentials-being-used
+  perf_test:
+    machine:
+      enabled: true
+      docker_layer_caching: true
```

Comment:
Do we need the `docker_layer_caching`? I don't see any docker commands. Same for the other use-sites